### PR TITLE
Issue 332: Move viewer package to larray-editor project 

### PR DESCRIPTION
**Pull request by alixdamman**
Currently, building of documentation crashes because pip does not find any package for larray-editor.

### CODE REVIEW

File: larray/viewer/__init__.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
 from __future__ import absolute_import, division, print_function
 
 try:
-    from larray.viewer.api import *
+    from larray_editor import *
 except ImportError:
     def view(*args, **kwargs):
-        raise Exception('view() is not available because Qt is not installed')
+        raise Exception('view() is not available because package larray_editor is not installed')
```
Comment (gdementen) [inline larray/viewer/__init__.py]: s/package larray_editor/the larray_editor package/

File: larray/viewer/__init__.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
     def edit(*args, **kwargs):
-        raise Exception('edit() is not available because Qt is not installed')
+        raise Exception('edit() is not available because package larray_editor is not installed')
```
Comment (gdementen) [inline larray/viewer/__init__.py]: idem

File: doc/source/changes/version_0_25.rst.inc [hunk 3]
```diff
@@ -1,6 +1,9 @@
 ﻿New features
 ------------
 
+* viewer function view, edit and compare require to have package larray-editor installed.
```
Comment (gdementen) [inline doc/source/changes/version_0_25.rst.inc]: I would write something like:

viewer functions (view, edit and compare) have been moved to the separate `larray-editor` package, which needs to be installed separately, unless you are using `larrayenv`.

File: doc/source/changes/version_0_25.rst.inc [hunk 4]
```diff
@@ -1,6 +1,10 @@
 ﻿New features
 ------------
 
+* viewer functions (`view`, `edit` and `compare`) have been moved to the separate `larray-editor` package,
+  which needs to be installed separately, unless you are using `larrayenv`.
+  Closes issue:`332`.
```
Comment (gdementen) [inline doc/source/changes/version_0_25.rst.inc]: the `:` before "issue" is important (so that it is picked up by sphinx).

Comment (alixdamman) [inline doc/source/changes/version_0_25.rst.inc]: Yes, indeed. Sorry.

**Comment by alixdamman** [comment]
updated

**Pull request merged**